source/system/Exception.ooc

```diff
@@ -200,9 +200,6 @@ OutOfBoundsException: class extends Exception {
 	init: func ~noOrigin (accessOffset, elementLength: Int) {
 		message = "Trying to access an element at offset %i, but size is %i!" format(accessOffset, elementLength)
 	}
-	kean_exception_outOfBoundsException_throw: static unmangled func (accessOffset, elementLength: Int) {
```

Comment:
This function is called from Array.h in safe version https://github.com/magic-lang/ooc-kean/blob/master/source/system/Array.h#L13 , make sure it works when building with -Dsafe.